scripts/CEE/cluster-health-check/run-oc-hc.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Download the lastest release of oc-hc and runs it 
+set -eou pipefail
+
+# Download latest version and move it to /usr/local/bin
+curl -sL "$(curl -s https://api.github.com/repos/givaldolins/openshift-cluster-health-check/releases/latest | grep browser_download_url | cut -d\" -f4 | grep -E 'linux-amd64.tar.gz$')" | tar zx
```

Comment:
Hi, sorry, we cannot run a remote script/binary here with the managed-scripts.
If you would like the same functionality, please put all the functions in the script itself.

Reply:
Hi @bmeng ,
That is too bad. I would have to refactor the whole thing. The idea behind doing it as an oc module is that we could recommend customers run that on their own and would use that on our side as a managed script.
Do you mind explaining the reason why we can't take that approach?
Also going an additional step, the code is in golang, could we use it directly instead of using the binary?